common-pg/src/main/scala/com/socrata/pg/soql/SoQLAnalysisSqlizer.scala

```diff
@@ -454,17 +454,30 @@ trait SoQLAnalysisSqlizer {
     // * select ... from single_row => select ...
     // * select ... from single_row join other_table on true .. => select ... from other_table ...
 
-    val (from, finalJoinPhrase, finalSetParamsJoin) =
-      if ((prevAna.isEmpty && fromTableName.isEmpty) || (fromTableName.map(_.name) == Some(TableName.SingleRow) && joinPhrase.isEmpty)) ("", joinPhrase, setParamsJoin)
-      else if (fromTableName.map(_.name) == Some(TableName.SingleRow) && joins.head.typ == InnerJoinType && joins.head.on == BooleanLiteral(true, SoQLBoolean)(joins.head.on.position)) {
-        (s" FROM ${joinPhrase.head.tableSql}", joinPhrase.tail, setParamsJoin.tail)
-      } else (s" FROM ${tableNamesWithThis(tableName.copy(alias = None))}" + tableName.alias.map(a => s" as ${a}").getOrElse(""), joinPhrase, setParamsJoin)
+    val (from, fromParams, joinPhrase, setParamsJoin) =
+      if ((prevAna.isEmpty && fromTableName.isEmpty) || (fromTableName.map(_.name) == Some(TableName.SingleRow) && joinPhrasesAndParams.isEmpty)) {
+        val (joinPhrases, joinSetParamses) = joinPhrasesAndParams.unzip
+        ("", Nil,
```

Comment:
this line break?

Reply:
separating from and its params with the joins and their params.